src/routes/Home.js

```diff
@@ -70,7 +75,7 @@ function Home () {
         if (remediations.status === 'fulfilled' && filter.value === filterText) {
             setShouldUpdateGrid(true);
         }
-    }, [ sorter.sortBy, sorter.sortDir, filter.value, pagination.pageSize, pagination.pageDebounced ]);
+    }, [ sorter.sortBy, sorter.sortDir, filter.value, pagination.pageSize, pagination.pageDebounced, showArchived ]);
```

Comment:
@Jason-RH is this **showArchived** supposed to be a hideArchived?

Reply:
Nope. A little confusing since we switched to using hideArchived on the backend, but showArchived is what gets changed when toggling the archive view.

Reply:
Ah!  I see, thank you for the clarification!